topics/cluster-spec.md

```diff
@@ -550,6 +581,16 @@ Slots migration is explained in similar terms but with different wording
 (for the sake of redundancy in the documentation) in the `CLUSTER SETSLOT`
 command documentation.
 
+Starting from Valkey 8.0, when the primary in either the source or target shard fails during live reconfiguration,
+the primary in the other shard will automatically attempt to update its migrating/importing state to correctly pair
+with the newly elected primary. If this update is successful, the ASK redirection will continue functioning without
+requiring administrator intervention. In the event that slot migration fails, administrators can manually resume
+the interrupted slot migration by running the command `valkey-cli --cluster fix <ip:port>`.
+
+Additionally, since Valkey 8.0, replicas are now able to return `ASK` redirects during slot migrations.
+This capability was previously unavailable, as replicas were not aware of ongoing slot migrations in earlier versions.
+It's worth noting that this change has already been documented in the READONLY command section.
```

Comment:
"already" and "worth noting", well, I added it a month ago, so roughly at the same time as this. We can just refer to READONLY if you want.

```suggestion
See the [READONLY](../commands/readonly.md) command.
```